src/table/src/engine.rs

```diff
@@ -104,7 +106,19 @@ pub trait TableEngine: Send + Sync {
     /// Drops the given table. Return true if the table is dropped, or false if the table doesn't exist.
     async fn drop_table(&self, ctx: &EngineContext, request: DropTableRequest) -> Result<bool>;
 
-    /// Close the table.
+    /// Closes the (partial) given table.
+    ///
+    /// Removes a table if all regions are closed.
+    /// Returns true if table is removed in the engine.
+    /// Returns false if table is only partial closed.
```

Comment:
I think we can do better by at least using an enum to indicate the table closing status. "boolean" is too narrow for this.